Dockerfile-k8s

```
@@ -0,0 +1,68 @@
+# TODO merge with Dockerfile when k8s branch is merged
+
+FROM ruby:2.7.6-slim-bullseye
+
+# Allow for setting ENV vars via --build-arg
+ARG BUNDLE_ENTERPRISE__CONTRIBSYS__COM \
+  RAILS_ENV=development \
+  USER_ID=1000
+ENV RAILS_ENV=$RAILS_ENV \
+  BUNDLE_ENTERPRISE__CONTRIBSYS__COM=$BUNDLE_ENTERPRISE__CONTRIBSYS__COM \
+  BUNDLER_VERSION=2.1.4
+
+RUN groupadd --gid $USER_ID nonroot \
+  && useradd --uid $USER_ID --gid nonroot --shell /bin/bash --create-home nonroot --home-dir /app
+
+WORKDIR /app
+
+RUN echo "deb http://ftp.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list
+RUN apt-get update
+RUN apt-get install -y -t testing poppler-utils
+RUN apt-get install -y build-essential libpq-dev git imagemagick curl wget pdftk file \
+  && apt-get clean \
+  && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Relax ImageMagick PDF security. See https://stackoverflow.com/a/59193253.
+RUN sed -i '/rights="none" pattern="PDF"/d' /etc/ImageMagick-6/policy.xml
+
+
+# Install fwdproxy.crt into trust store
+# Relies on update-ca-certificates being run in following step
+COPY config/ca-trust/*.crt /usr/local/share/ca-certificates/
+
+# Download VA Certs
+COPY ./import-va-certs-pe.sh .
```

Since we are de-duping this cert script this needs to change --

```suggestion
COPY ./import-va-certs-k8s.sh .
```